### PR TITLE
Project deployment

### PR DESCRIPTION
**Pull request by DamianDib9**
### **_Project requirements_**
_________________

- Deploy your website using GitHub Pages. (I had to deploy it with Netlify because Github pages had issues on changes sync)✔️
- Check the online version of your portfolio and make sure that the page works properly.✔️
- Update the README of your repository to include a link to the online version.✔️
- If by any chance you have deployed your app earlier, open an issue with a link to the online version of your app. Otherwise, submit a link to the PR with changes required for the deploy.✔️

Hope you like it! 😃👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,9 +60,10 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- <p align="right">(<a href="https://mellow-cassata-728b9c.netlify.app/">Click here</a>)</p>
+- To see live Demo  <a href="https://mellow-cassata-728b9c.netlify.app/">Click here</a></p>
```
Comment (abdulhamiid) [inline README.md]: - According to the project requirement, please deploy your website using **GitHub Pages** and update the README of your repository to include a link to the online version. 👍 😃 

image

- Changes made to a GitHub Pages page do not appear as they are made, as GitHub uses CDN caching for Pages, and there is a small delay for changes to be reflected on the actual page. 😊

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @DamianDib9 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Deploy App
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider the following:
- x
- y
- z

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (DamianDib9) [reply README.md]: OK, done 👍

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @DamianDib9    ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


Highlights
- [ ] Good code organization ✔️
- [ ] Nice use Github pages for deployment ✔️ 

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**